apicast/http.d/lua_capture_error_log.conf

```diff
@@ -0,0 +1 @@
+lua_capture_error_log 4k;
```

Comment:
This is enough 10 log entries.
So it will collect last 10 log entries with desired log level. 

For example when you have between prometheus pulls 15 log entries like: 5 error, 10 warning then the error will not appear.
It is always good to set the desired log level to something we actually need.

Reply:
So maybe this conf could be an env var?  Because we certainly will have to play with the Prometheus scrape time and the log capture size.

Reply:
It could be when it would be part of the liquid template in the main repo.
But here it could not be templated.

And disregard my comment about 10 entries. It fits more.

Well I think 4k is fine when properly configured. We should not capture log levels we don't care about. So lets say there are top ones: emerg, alert, crit, error.
And we configure to capture `error` and up. We would not really care if there are some missing higher levels, because the error itself is enough to trigger a warning.

Reply:
And this is controlled by the "log_map" var? can we expose this one as ENV ?

Reply:
I see so `set_filter_level`  will capture everything `>=METRICS_LOG_LEVEL` right?

Reply:
@jmprusi done in https://github.com/3scale/apicast-cloud-hosted/pull/5/commits/f45985ced7236270dee6e9fdaaebf37c29771ba5 as `METRICS_LOG_LEVEL`. better name suggestions much welcome :) 

@maneta yes.

Reply:
Cool! :)